tools/config/winlogbeat.yml

```diff
@@ -203,8 +203,6 @@ fieldmappings:
     ContextInfo: winlog.event_data.ContextInfo
     # from here missing field at 20210706
     Accesses: winlog.event_data.Accesses
-    AccessList: winlog.event_data.AccessList
```

Comment:
Why did you remove these field mappings?

Reply:
Duplicate keys invalidate YAML

Reply:
sorry my bad :confounded: